kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/device/SystemPropertiesTabUi.java

```diff
@@ -146,4 +146,8 @@ public void onSuccess(ArrayList<GwtGroupedNVPair> result) {
         });
     }
 
+    @Override
```

Comment:
Please change the header to: " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates"